src/main/resources/application.properties

```diff
@@ -16,10 +16,8 @@ spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.open-in-view=false
 
 # Kafka
-kafka.topic=callisto-timecard
-
 spring.kafka.bootstrap-servers=${BOOTSTRAP_SERVER}
-spring.kafka.template.default-topic=callisto-timecard
+spring.kafka.template.default-topic=${TIMECARD_TOPIC}
```

Comment:
worth using a default/fallback value for local use?
E.g. `spring.kafka.template.default-topic=${TIMECARD_TOPIC:callisto-timecard-accruals}`

Reply:
I am concerned about having a default for local dev currently. That is mainly because the topic name for local dev is different. So we could end up the scenario where we build a topic with an incorrect name in the namespace.

We could however make sure local dev and timecard match. Therefore removing the above issue?

Reply:
Ah I didn't realise localdev had a different topic name.
I think ideally yes it would be best to have them match, but only if it's not too much extra effort/testing.
Otherwise we can leave this.